packages/address/src/tests/utilities.test.ts

```diff
@@ -1,10 +1,18 @@
+import {Address} from '@shopify/address-consts';
 import {fixtures} from '@shopify/address-mocks';
 
-import {renderLineTemplate} from '../utilities';
+import {
+  buildOrderedFields,
+  formatAddress,
+  renderLineTemplate,
+} from '../utilities';
 
 const Canada = fixtures.country.EN.data.country;
+const Japan = fixtures.countries.JA.data.countries.find(
+  ({code}) => code === 'JP',
+)!;
```

Comment:
I'm using a non-null assertion here, though I'm not a fan of this, since it's tests based on fixtures, I figured the risk was minimal.